js-miniapp-sdk/src/types/responseTypes/interstitial/index.ts

```diff
@@ -0,0 +1,7 @@
+import { AdTypes } from '../../adTypes';
+/**
+ * Intterstitial ad type response from native sdk
+ */
```

Comment:
```suggestion

/**
 * Interstitial ad type response from native SDK
 */
```

Reply:
The documentation could utilize aspects of the purpose of this interface, intended usage, scope etc.